data/json/items/armor/power_armor.json

```diff
@@ -1,694 +1,670 @@
+      "type": "transform",
+      "msg": "Exoskeleton boot sequence initiated\nLiquid cooling system engaged\nAtmospheric scrubber system engaged\nMotors engaged\nBoot sequence terminated, all systems nominal.",
+      "target": "power_armor_basic_on",
+      "active": true,
+      "need_charges": 1,
+      "need_charges_msg": "The exoskeleton batteries are empty"
+    }
   },
   {
     "id": "power_armor_basic_on",
     "copy-from": "power_armor_basic",
-    "looks_like": "power_armor_basic",
     "type": "TOOL_ARMOR",
+    "looks_like": "power_armor_basic",
     "name": { "str": "combat exoskeleton (on)", "str_pl": "combat exoskeletons (on)" },
-    "description": "These were the second wave of military combat exoskeleton, and got a lot of media attention, with popular Navy commercials featuring them heavily.  It consists of a muscle-boosting exoskeleton frame with overlaid segmented alloy plating.  Despite advancements over the original bulky 'tank suits', the wearer still cannot easily fit through narrow spaces, or sit down comfortably (and it ruins upholstery).  There is an integrated chemical-resistant bodyglove that precludes wearing other clothing.  It is turned on and continually drawing power.  Use it to turn it off.",
-    "flags": [ "USE_UPS", "WATERPROOF", "STURDY", "ELECTRIC_IMMUNE", "TRADER_AVOID", "COMBAT_TOGGLEABLE", "UNBREAKABLE_MELEE" ],
-    "power_draw": "4 kW",
+    "description": "This combat exoskeleton is of second generation, designed to be lighter, less encumbering and more energy efficient while still giving the user greatly increased carry capacity. The exoskeleton is actually composed of a form-fitting liquid cooling garment, a protective layer reminiscent of a flight suit, enclosed by a transparent mask, and the exoskeleton itself, micro motors and actuators on a complex frame with hooks and straps to attach specialized armor onto. The exoskeleton is on and its components whirring softly as you move.",
+    "power_draw": "84 kW",
+    "weight_capacity_modifier": 1.6,
     "revert_to": "power_armor_basic",
     "use_action": { "type": "transform", "menu_text": "Turn off", "msg": "The %s armor disengages.", "target": "power_armor_basic" },
     "armor": [
       {
+        "layers": [ "SKINTIGHT", "NORMAL", "OUTER" ],
+        "covers": [ "head", "torso", "arm_l", "arm_r", "hand_l", "hand_r", "leg_l", "leg_r", "foot_l", "foot_r" ],
         "encumbrance": 10,
         "coverage": 100,
-        "covers": [ "torso", "arm_l", "arm_r", "hand_l", "hand_r", "leg_l", "leg_r", "foot_l", "foot_r" ]
+        "material": [
+          { "type": "lycra", "thickness": 0.2 },
+          { "type": "nomex", "thickness": 0.8 },
+          { "type": "steel", "thickness": 25, "covered_by_mat": 10 }
+        ]
+      },
+      {
+        "layers": [ "NORMAL" ],
+        "covers": [ "mouth", "eyes" ],
+        "encumbrance": 0,
+        "coverage": 100,
+        "material": [ { "type": "lvl4ballisticglass", "thickness": 2 } ]
```

Comment:
Did you test just using proportional encumbrance instead of rewriting the "armor"?

Reply:
@Procyonae I have no idea how to use "proportional" for encumbrance. I don't even know if proportional works for values that are inside an array. There is no armor item that does that in fact unless I'm mistaken, all values that are proportionally modified in a child are values that aren't in arrays, like "weight", "price", stuff like that.